### PR TITLE
[playground] Update docker based deployment

### PR DESCRIPTION
**Pull request by theBeginner86**
**Notes for Reviewers**

This PR documents recent changes done to the docker based deployment of [Cloudnative Playground](https://playground.meshery.io).

Changes:
1. Added missing watch label to all services so that watchtower can update containers of each service defined in docker-compose.
2. Reduced the [Polling interval](https://containrrr.dev/watchtower/arguments/#poll_interval) to 6hrs for watchtower service so that the frequency by which it pulls latest images increases. Reason being that by default watchtower pulls the latest image in 24 hrs which means if a release was made then playground deployment would be updated in 24hrs and not immediately.
3. Documented the complete nginx config. Earlier we were just documenting part of the config in `/etc/nginx/conf.d/default.conf` which was being imported into `/etc/nginx/nginx.conf` (default location). So also updated the container mount for nginx service.
4. Exposed port 80 for nginx service so that HTTP to HTTPS redirection works.

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: install/playground/docker/nginx.conf [hunk 1]
```diff
@@ -25,7 +64,7 @@ server {
 	   proxy_pass http://meshery-meshery-1:9081;
```
Comment (Revolyssup) [inline install/playground/docker/nginx.conf:64]: Is the uri correct here? It wont be http://meshery:9081 ?

Comment (MUzairS15) [reply install/playground/docker/nginx.conf:64]: Yes, as the meshery container gets named meshery-meshery-1, (i.e. network-container name-serial no.)

**Review by MUzairS15 — COMMENTED** [review]
Reducing the polling time to 6hrs from 24 hrs is helpful. But I will say, it will be nice if we update the release workflow to pull the latest image and deploy it again.

**Comment by theBeginner86** [comment]
> Reducing the polling time to 6hrs from 24 hrs is helpful. But I will say, it will be nice if we update the release workflow to pull the latest image and deploy it again.

Sounds good. 
This caveat of watchtower was the reason we moved away from it in remote provider . I'll create an issue to follow up on this.

**Pull request merged**